dbfit-java/informix/src/integration-test/resources/deinstall_informix.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+# Run as root to deinstall Informix and any existing database server.
+# Exit statuses: -
+# 0 - deinstalled successfully.
+# 1 - error occured.
+MODNAME=`basename $0`
+IM="$MODNAME: INFO:"
+EM="$MODNAME: ERROR:"
+WM="$MODNAME: WARNING:"
+
+DBFIT_ROOT=/var/dbfit
+INFORMIXDIR=/informix
+INFORMIX_SCRIPTS=$DBFIT_ROOT/dbfit-java/informix/src/integration-test/resources
+
+if [ ! -x "$INFORMIXDIR/uninstall/uninstall_ids" ]
+then
+	echo "$EM cannot find Informix deinstaller $INFORMIXDIR/uninstall/uninstall_ids" >&2
+	exit 1
+fi
+
+. $INFORMIXDIR/dbfitifserver.ksh
```

Comment:
This shell script is created by the Informix installer. It just sets and exports some environment variables so that the tools can be used and looks fine to use with `bash`: -

```
INFORMIXDIR=/informix
INFORMIXSERVER=dbfitifserver
ONCONFIG=onconfig.dbfitifserver
INFORMIXSQLHOSTS=/informix/etc/sqlhosts.dbfitifserver
GL_USEGLU=1
PATH=${INFORMIXDIR}/bin:${INFORMIXDIR}/extend/krakatoa/jre/bin:${PATH}
export INFORMIXDIR INFORMIXSERVER ONCONFIG INFORMIXSQLHOSTS GL_USEGLU PATH